src/TouristInfoCards.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+// import TouristInfoCards from TouristInfoCards;
+
+// const TouristInfoCards = () => {
+function TouristInfoCards() {
+  return (
+    <div className="card-container">
```

Comment:
For this component, you are repeating one functionality three times, is there a better way of doing so that we've learned in JS module?